lib/fieldhand/paginator.rb

```diff
@@ -11,20 +11,25 @@ module Fieldhand
   #
   # See https://www.openarchives.org/OAI/openarchivesprotocol.html#FlowControl
   class Paginator
-    attr_reader :uri, :logger, :timeout, :headers, :http
+    attr_reader :uri, :logger, :timeout, :interval, :headers, :http
+    attr_accessor :retries
```

Comment:
any reason why retries is specifically an accessor (= you want write access outside of the initialize)?

Reply:
It's in order to decrease the value in `retries -= 1`